docs/source/resources/releases_maintenance.rst

```diff
@@ -89,6 +89,11 @@ The z/OS managed node includes several shells, currently the only supported shel
 +---------+----------------------------+---------------------------------------------------+---------------+---------------+
 | Version | Controller                 | Managed Node                                      | GA            | End of Life   |
 +=========+============================+===================================================+===============+===============+
+| 1.11.x  |- `ansible-core`_ >=2.15.x  |- `z/OS`_ V2R4 - V2Rx                              | In preview    | TBD           |
```

Comment:
Weren't we supposed to have z/OS 3.1 support for the beta?
```
`z/OS`_ V2R4 - V3Rx
```

Reply:
you're right, fixed